data/transport missions.txt

```diff
@@ -39,6 +39,36 @@ mission "Immigrant Workers"
 				accept
 
 
+mission "Vintage Spaceman"
+	description "A young man wants to find love on <destination>."
+	minor
+	passengers 1
+	to offer
+		random < 50
+	source "X"
+	destination
+		attributes core
+	on complete
+		payment 10000
+		conversion
+			`	"We're here!" Vintage spaceman slams the emergency unlock and jumps out while your ship is still a meter off the ground. He runs off in to the crowd.`
+			`	You grab a drink, climb on top of the <ship> and relax.`
+			`	After the concert is over you see your passenger emerge from the crowd, his head hung low. "It's all a ruse man. She was just an bot. A marketing tool to get me to buy tickets."`
+			`	"Guess she *was* made for you." You quip, feeling a little bad about it.`
+			`	"Sorry I wasted your time. I'm such a sucker. I don't have any credits for you, but you can have this suit. I don't feel like wearing it any more."`
+			`	He leaves the suit on your deck and disappears back in to the crowd. You take the suit to a pawn shop and get X credits for it.`
+
+	on offer
+		conversation
+			`	"Waaaaazzzup?" A patron in his mid twenties approaches after you accidently make eye contact. "Dude." He continues. "That your ship? You gotta help me. I'm in love!"`
+			`	He's dressed in a vintage spacesuit, something they wore before space travel was common. Useless these days, but a rare collector's item.`
+			`	"It all started last week. I was in the Spritney Beers chat room and started talking to this rad chick. We took it to DM and talk every day. It's like she was made for me. I need to see her IRL. She invited me to the next Spritney Beers concert on <system>. You have to take me!"`
+			choice
+				`	"Fine, but I'm in control of the music."`
+					accept
+				`	(Point like there's someone famous behind him and make a run for the door.)`
+					decline
```

Comment:
I figure DM is probably 'direct message', used particularly on Twitter. I'm not sure if it's appropriate in a sci-fi, but it'd be nice to decide some kind of consistent lingo for some of this stuff that we can use everywhere in the game.